inc/packages/namespace.php

```diff
@@ -593,29 +593,38 @@ function get_update_data( $did ) {
 	$required_versions = version_requirements( $release );
 	$filename = get_hashed_filename( $metadata );
 	$type = str_replace( 'wp-', '', $metadata->type );
+	$sections = (array) $metadata->sections;
+	$description = trim( $sections['description'] ) ?? '';
```

Comment:
Need to trim the result of `$sections['description'] ?? ''` instead.

Trimming `$sections['description']` when it doesn't exist would throw a warning, and `trim()` returns a string, so `??` would never be hit.

```php
$description = trim( $sections['description'] ?? '' );
```